data/json/mapgen/park.json

```diff
@@ -401,14 +401,14 @@
       "rows": [
         "zzzzz qq zzzzzzzzzzzzzzz",
         "zfffz qq z             z",
-        "zftfz qq z t  bTb    t z",
-        "zfffz qq zz   bTb      z",
+        "zftfz qq z t  cTc    t z",
+        "zfffz qq zz   cTc      z",
         "zfffz qq! zz       !   z",
         "zfffz qqq  zz          z",
         "zftfz  qqq,,,,,        z",
-        "zfffzz  qqq zz &   bb  z",
+        "zfffzz  qqq zz &   cc  z",
         "zzzzzzz  qq  z     TT  z",
-        "         qqq zz    bb  z",
+        "         qqq zz    cc  z",
```

Comment:
So below you make the b furniture a park bench and then up here in the park you make the mapgen use c which is a non park bench?

Reply:
Indeed! This is partially because I am an idiot and partially because I didn't notice this not only added more bloat but didn't even resolve the issue. I'll fix it in a sec.